.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -44,6 +44,7 @@ _Any other detail that may help to understand/debug the problem_
 
 ### Checklist
 
+- [x] This is a checked item.
```

Comment:
:+1: for this.

I am not sure if this is the prettiest solution (having an own ugly example checkitem). Maybe something like

```markdown
<!--
- [x] This is a checked item.
-->
```

```markdown
### Checklist (check them like this `- [x]`)
```

Maybe it would make sense to link to https://guides.github.com/features/mastering-markdown/ too. WDYT?

Reply:
@rusty-snake [on Sep 19](https://github.com/netblue30/firejail/pull/4561#discussion_r711730650):

> +1 for this.
> 
> I am not sure if this is the prettiest solution (having an own ugly example
> checkitem).

Certainly not a pretty solution, though it's the most dummy-proof one that I
could think of.  And considering the formatting of some issues, dummy-proofing
seems like a helpful quality...

> Maybe something like
> 
> ```gfm
> <!--
> - [x] This is a checked item.
> -->
> ```

I had thought about doing this, but it seemed a bit less obvious and it takes
more space.  If it works, it's good enough though.  Added it.

> Maybe it would make sense to link to
> https://guides.github.com/features/mastering-markdown/ too. WDYT?

Done.